site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -1,6 +1,56 @@
 ---
 api: runtime
 ---
+The runtime API provides methods to support a number of areas of functionality that your extensions
+can use:
+
+Message passing
+
+: These methods support message passing between tasks within the extension (such as background
+  scripts) and with the native system on which the browser is running. See [Message
```

Comment:
To me, this almost reads as if this only supports passing messages between extension bits and the native system, rather than intra-extension tasks *and* the system.  Also, messaging can be used to communicate with other extensions.  The intra-extension messaging is probably most popular.  What about:

```suggestion
: These methods support message passing so that you can communicate with different parts of your extension (such as an extension popup and background scripts), other extensions, or native applications on the user's device. See [Message
```